TheSteelGuy/Ride-My-WayApiV1
# Ft failing testcases #158660764

**What does this PR do**
creates all test cases for challenge 3 endpoints
**Description of Task to be completed**

write new tests for challenge 3 endpoints

**How should I test this**

- Create a virtualenv and activate it:
- Clone the repo
- cd into the cloned repo
- switch to branch ft-failing-testcases#158660764

Install the dependencies::

pip install -r requirements.txt 
do pytest -cov=tests
NB they should all fail
